usim/__about__.py

```diff
@@ -32,7 +32,7 @@
 __summary__ = 'Lightweight Simulation Framework'
 __url__ = 'https://github.com/MaineKuehn/usim'
 
-__version__ = '0.2.0'
+__version__ = '0.2.1'
```

Comment:
As this is not just a minor fix I would propose setting the version to 0.3.0.

```suggestion
__version__ = '0.3.0'
```